src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+
```

Comment:
[Don't begin class or method implementation with an empty line.](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-begin-class-or-method-implementation-with-an-empty-line)

Check all places

Reply:
it's for checkstyle, otherwise project won't build